src/Slim/Middleware/HandlerTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Slim\Middleware;
+
+use SessionHandlerInterface as Handler;
+
+trait HandlerTrait
+{
+    protected function registHandler(Handler $handler)
+    {
+        if (PHP_VERSION_ID >= 50400) {
+            session_set_save_handler($handler, true);
+        } else {
```

Comment:
We depend on Slim 3 (PHP 5.5+), so even the comparison is not needed at all.

Reply:
It's OK, I will remove it.

Reply:
Done